src/main/kotlin/racingcarWinner/domain/Cars.kt

```diff
@@ -0,0 +1,21 @@
+package racingcarWinner.domain
+
+import racingcarWinner.util.MessageCode
+
+class Cars(carNames: List<String>) {
+    val carList: List<Car> = carNames.map { Car(it.trim()) }
```

Comment:
```suggestion
    private val carList: List<Car> = carNames.map { Car(it.trim()) }
```
여기도 외부에서 접근할 수 없도록 해주세요 :)